README.md

```diff
@@ -62,6 +62,10 @@ Once toolchain and other dependencies are in place, run `make build` to build, a
 `stratisd` executable in `./target/x86_64-unknown-linux-gnu/debug` as root.
 Pass the `--help` option for more information on additional developer options.
 
+##### Building tests
+The Makefile provides a target, `build-tests` which allows compiling the
+tests without running any of them, as a convenience to developers.
```

Comment:
Just wondering: shouldn't it be "as a convenience **for** developers"? If there is a rule/exception about this in english, I'd really like to learn it.

Reply:
Either is correct. There's probably a very subtle difference in meaning. Something like "convenient to" would make more sense when thinking of a tool that could be grabbed quickly (probably my thought at the time). "convenient for" would emphasize the person and their experience more.

Reply:
I see, thanks for the explanation!